rpc.go

```diff
@@ -18,18 +18,18 @@ type CallData struct {
 // RPCCall executes a remote procedure call on the node.
 //
 // This method should be used by generated code only.
-func (n *RawNode) RPCCall(ctx context.Context, d CallData) (resp protoreflect.ProtoMessage, err error) {
+func (n *RawNode) RPCCall(ctx context.Context, d CallData) (protoreflect.ProtoMessage, error) {
 	md := &ordering.Metadata{MessageID: n.mgr.getMsgID(), Method: d.Method}
 	replyChan := make(chan response, 1)
 	n.channel.enqueue(request{ctx: ctx, msg: &Message{Metadata: md, Message: d.Message}}, replyChan, false)
 
 	select {
 	case r := <-replyChan:
 		if r.err != nil {
```

Comment:
Here we can simply return without the proceeding `if`:
```go
return r.msg, r.err
```
Or did I miss something? The tests still pass with this version.